package.json

```diff
@@ -61,11 +62,14 @@
     "babel-plugin-transform-flow-strip-types": "^6.8.0",
     "babel-preset-es2015": "^6.14.0",
     "babel-register": "^6.14.0",
+    "compat-table": "github:kangax/compat-table#gh-pages",
```

Comment:
In this form, it's definitely unsafe. We can have breaking changes. Required https://github.com/kangax/compat-table/issues/711